common/src/main/java/org/opensearch/ml/common/connector/Connector.java

```diff
@@ -93,7 +93,7 @@ static Connector fromStream(StreamInput in) throws IOException {
     }
 
     static Connector createConnector(XContentBuilder builder, String connectorProtocol) throws IOException {
-        String jsonStr = Strings.toString(builder);
+        String jsonStr = StringUtils.xContentBuilderToString(builder);
```

Comment:
Can't we do : `builder.toString()`. Core also did the same: https://github.com/opensearch-project/OpenSearch/commit/d090a7f1e8be0df161f3d48f264bd0943766417e

Reply:
Yup thats the right way to do it. 
The static method was refactored and moved to `XContentBuilder`